esm_runscripts/batch_system.py

```diff
@@ -192,6 +193,32 @@ def get_extra(config):
             extras.append("source "+config["general"]["experiment_dir"]+"/.venv_esmtools/bin/activate")
         if config["general"].get("funny_comment", True):
             extras.append("# 3...2...1...Liftoff!")
+        # Search for ``pre_run_commands``s in the components
+        for component in config.keys():
+            pre_run_commands = config[component].get("pre_run_commands")
```

Comment:
does this mean that pre_run_commands cannot go into the `general:` section?

Reply:
It can go in the `general` section. I think the `component` name is misleading, config contains `general`, `computer`, `foci` ... so we are looping through `general` too.